### ISSUE TITLE
[FEATURE] Store the session ID or use cookies or JWT to prevent user logging out after refreshing.

### ISSUE THREAD

**Issue by promehul** [HelikarLab/candis#168]
**Describe the bug**
When I log in and refresh the page , user is logged out and we come back to the log in page.

**Expected behavior**
User shouldn't be logged out

**Additional context**
This is happening because we are no where saving the sessions id on the browser.

**Comment by promehul** [comment]
@achillesrasquinha I dont know whether this is a feature intentionally implemented because of security reasons or is it really a bug ?

**Comment by rupav** [comment]
We are using https://github.com/marcuswestin/store.js/ to store the credentials, username and tokenId in the browser, which gets erased after a refresh.

**Comment by promehul** [comment]
So this is not a bug ,rather a feature na?

**Comment by rupav** [comment]
Yes, not a bug, rather an implementation of store.js, which stores it in local storage if found. For persisting the token on a refresh, we have to check other options like cookies.
I Don't think currently this feature is of high priority.

**Comment by pranay-simejia** [comment]
Is this issue open? I could work on storing user credentials on browser cookies.
